simmechanics/data/icub2_5/ICUB_2-5_BB_simmechanics_options.yaml.in

```diff
@@ -546,6 +546,18 @@ exportedFrames:
   - frameName: SCSYS_L_FOOT_SKIN_22
     frameReferenceLink: l_foot
     exportedFrameName: l_sole_skin_22
+  - frameName: SCSYS_R_UPPER_LEG_FT-IMU_3B11
+    frameReferenceLink: r_hip_2
+    exportedFrameName: r_upper_leg_ft_imu_3B11_frame
+  - frameName: SCSYS_R_FOOT_FT-IMU_3B14
+    frameReferenceLink: r_ankle_2
+    exportedFrameName: r_foot_ft_imu_3B14_frame
+  - frameName: SCSYS_L_UPPER_LEG_FT-IMU_3B12
+    frameReferenceLink: l_hip_2
+    exportedFrameName: l_upper_leg_ft_imu_3B12_frame
+  - frameName: SCSYS_L_FOOT_FT-IMU_3B13
+    frameReferenceLink: l_ankle_2
+    exportedFrameName: l_foot_ft_imu_3B13_frame
```

Comment:
As far as I remember, sensors don't need an exported fixed URDF frames, i.e. fake link with zero mass connected to a link with a fixed joint. this is automatically handled by iDyntree, the `exportFrameInURDF` parameter is not mandatory (refer to https://github.com/robotology/simmechanics-to-urdf/blob/master/README.md).